src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1464,6 +1469,7 @@ def generate(
         task=None,
         language=None,
         is_multilingual=None,
+        prompt_ids: Optional[Union[torch.Tensor, np.ndarray]] = None,
```

Comment:
Sorry to be a stickler about this one, but why was `np.ndarray` added back in here?

Reply:
No worries, I thought it was implied by @sanchit-gandhi 's ask to have the default return_tensors of get_prompt_ids be 'np'. Should I take it out?

Reply:
I'll let Sanchit have the final say on this then. 😉

Reply:
Got it thanks, put the type back to being just pytorch